perllib/FixMyStreet/App/Controller/Admin/Stats.pm

```diff
@@ -72,4 +72,24 @@ sub questionnaire : Local : Args(0) {
     return 1;
 }
 
+sub refused : Local : Args(0) {
+    my ($self, $c) = @_;
+
+    my $contacts = $c->model('DB::Contact')->not_deleted->search({ email => 'REFUSED' }, { prefetch => 'body' });
```

Comment:
Should this also not cover devolved contact with the send_method set to Refused?